src/main/java/com/github/shoothzj/dev/dump/DumpAction.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.github.shoothzj.dev.dump;
+
+import com.github.shoothzj.dev.constant.K8sCmdConst;
+import com.github.shoothzj.dev.storage.StorageSettings;
+import com.github.shoothzj.dev.util.SshClient;
+import io.netty.util.internal.StringUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.UUID;
+
+public class DumpAction {
+
+    private static final Logger log = LoggerFactory.getLogger(DumpAction.class);
+
+    public String dump(String host, int port, String username, String password, String podName, String namespace, String pid) {
+        try {
+            SshClient sshClient = new SshClient(host, port, username, password);
+            List<String> VMArch = sshClient.execute(K8sCmdConst.execPodCmd(namespace, podName, "uname -m"), 5);
+            if (VMArch.isEmpty()) {
+                return "get vm arch failed.";
+            }
+            List<String> execute = sshClient.execute(K8sCmdConst.execPodCmd(namespace, podName, "java -version"), 5);
```

Comment:
please use $JAVA_HOME/bin/java -version

Reply:
$JAVA_HOME/bin/java -version can not use